packages/shared/lib/services/connection.service.ts

```diff
@@ -1,4 +1,5 @@
 import jwt from 'jsonwebtoken';
+import { Builder, parseStringPromise } from 'xml2js';
```

Comment:
can you check other options for the package? this one seems badly maintained, maybe those 2 can do the same:
- https://www.npmjs.com/package/fast-xml-parser (this one has large sponsor)
- https://www.npmjs.com/package/@xmldom/xmldom

Reply:
updated with the latest commit 👍.